airbyte-integrations/connectors/source-okta/source_okta/manifest.yaml

```diff
@@ -12,6 +12,10 @@ definitions:
     token_refresh_endpoint: "https://{{ config['domain'] }}.okta.com/oauth2/v1/token"
     grant_type: refresh_token
 
+  custom_oauth_private_key_authenticator:
+    type: CustomAuthenticator
+    class_name: source_okta.custom_authenticators.CustomOauth2PrivateKeyAuthenticator
```

Comment:
When you rename the file, make sure to change the reference here as well.

Reply:
Done - renamed in `manifest.yaml` and test files.